Dockerfile

```
@@ -1,11 +1,31 @@
+# Use Node 18
 FROM docker.io/library/node:18 as react-build
+
+# Use default answers in installation commands
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Copy cluster component source code
 WORKDIR /code
 COPY . /code
-# hadolint ignore=DL3003
-RUN cd reana-ui && \
+
+# Build frontend application
+# hadolint ignore=DL3003,DL3008
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y \
+      build-essential \
+      libcairo2-dev \
+      libgif-dev \
+      libjpeg-dev \
+      libpango1.0-dev \
+      librsvg2-dev && \
+    cd reana-ui && \
     yarn && \
```

Locally I see some network timeouts, probably due to emulation slowness. This fixes it:
```suggestion
    yarn --network-timeout 600000 && \
```